### DISCUSSION TITLE
Error in getting response from dicom web server

### DISCUSSION THREAD

**Discussion by athanell** [lassoan/SlicerDICOMwebBrowser#4] [Q&A] [answered]
Hello @lassoan ,

I've installed the DICOMwebBrowser extension in 3D Slicer (4.13.0-2021-02-17 r29718 / 5ca36fd),  used the server url mentioned in the readme.md and I was able to connect normally and browse through the different records. 

When though I tried to "download and index" a particular series I got into the following error that you can see in the following video link:

[Error in getting response from dicom web server](https://www.dropbox.com/s/3jvdn8rl9a3ryl9/SlicerDICOMWeb.mp4?dl=0)

What could possibly be the reason for that error?

Regards
Antonios

**Comment by lassoan** [comment, marked as answer]
It is not an error in the client, but an internal server error, so you need to ask the server administrators about it.

Since  https://server.dcmjs.org/dcm4chee-arc/ is a test server, it is normal that there are errors. It may be even intentional, so that they can test error cases.

**Reply by athanell** [reply]
Thank you @lassoan . One more question on the topic, since I get a client error when connecting to an azure dicom webserver.  I can see normally all the records through Slicer but when I try to download a series I get:
``` bash
DownloadSelectedSeries: Error in getting response from DICOMweb server.
HTTP Error:
406 Client Error: Not Acceptable for url: https://xxx-xxx.azurewebsites.net/studies/1.2.826.0.1.xxxxxx.x.xxx.x.x.xxxxxxxxxxx.xxxx.xxxx/series/1.2.826.0.1.xxxxxx.x.x.x.x.xxxx.xxx.xxxxx.xxxxxx/instances/1.2.826.0.1.xxxx.x.xxx.x.x.8xxxxx.xxx.xxxxxxxx.xxxxx
```
Does this mean that there is an issue with the retrieval of the serie's DICOM instances (WADO)? the data itself (non conformant DICOMS)? something else?

Antonios

**Reply by lassoan** [reply]
This appears to be a completely different problem. To keep each discussion focused, could you post this question as a new discussion? Thank you.

**Reply by athanell** [reply]
Yes I will. Thanks again @lassoan
